### PR TITLE
Update ODK to v1.3.0

### PR DESCRIPTION
**Pull request by anitacaron**
I ran `make test` locally, and everything passed successfully.

### CODE REVIEW

File: .github/workflows/qc.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Basic ODK workflow
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
```
Comment (matentzn) [inline .github/workflows/qc.yml:9]: wrong branch name. (`master`)

File: .github/workflows/qc.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  pull_request:
+    branches: [ main ]
```
Comment (matentzn) [inline .github/workflows/qc.yml:11]: wrong branch name. (`master`)

File: .github/workflows/qc.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    container: obolibrary/odkfull:v1.3.0
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Run ontology QC checks
+        env:
+          DEFAULT_BRANCH: main
```
Comment (matentzn) [inline .github/workflows/qc.yml:31]: wrong branch name. (`master`)

File: src/sparql/SCAtlas_terms.sparql [hunk 4]
```diff
@@ -3,5 +3,5 @@ WHERE {
   { ?s1 ?p1 ?term . }
```
Comment (matentzn) [inline src/sparql/SCAtlas_terms.sparql:3]: The fact that the finalname here has casing suggests a bug in ODK - maybe make an issue at ODK to ensure filenames using the ontology ID should be cycled through "lower". I noticed this also for the release artefacts..

Comment (matentzn) [reply src/sparql/SCAtlas_terms.sparql:3]: alternatively change the id in the scao rather than SCAtlas?

**Review by matentzn — CHANGES_REQUESTED** [review]

File: src/sparql/SCAtlas_terms.sparql (hunk 4 above)
Comment (anitacaron) [reply src/sparql/SCAtlas_terms.sparql:3]: the id in the yaml file is already scao

Comment (anitacaron) [reply src/sparql/SCAtlas_terms.sparql]: it worked! Thanks

**Review by matentzn — APPROVED** [review]
Nice work!

**Pull request merged**